### PR TITLE
Add blog post for mixit17 registration

### PR DESCRIPTION
**Pull request by cexbrayat**
Also add the text for the `/attend` page.

### CODE REVIEW

**Comment by cexbrayat** [comment]
This is to get the ball rolling, feel free to improve

File: src/main/resources/data/blog.json [hunk 1]
```diff
@@ -1,5 +1,47 @@
...
+      },
+      "headline": {
+          "ENGLISH": "MiXiT will open its ticket sale soon, but you need to pre-register first. Read more to know why.",
+          "FRENCH": "MiXiT ouvre bientôt sa billeterie mais vous devez vous pré-enregistrer d'abord. Cet article vous dit pourquoi."
+      },
+      "id": "314",
+      "slug": {
+          "ENGLISH": "attend-mixit17",
+          "FRENCH": "assister-mixit17"
```
Comment (clacote) [inline src/main/resources/data/blog.json]: `billetterie-mixit17` serait plus parlant

File: src/main/resources/data/blog.json [hunk 2]
```diff
@@ -1,5 +1,47 @@
...
+      },
+      "headline": {
+          "ENGLISH": "MiXiT will open its ticket sale soon, but you need to pre-register first. Read more to know why.",
+          "FRENCH": "MiXiT ouvre bientôt sa billeterie mais vous devez vous pré-enregistrer d'abord. Cet article vous dit pourquoi."
+      },
+      "id": "314",
+      "slug": {
+          "ENGLISH": "attend-mixit17",
```
Comment (clacote) [inline src/main/resources/data/blog.json]: `mixit17-tickets` or `mixit17-ticketing` would be more obvious

File: src/main/resources/data/blog.json [hunk 3]
```diff
@@ -1,5 +1,47 @@
...
+          "FRENCH": "assister-mixit17"
+      },
+      "title": {
+          "ENGLISH": "MiXiT pre-registration",
```
Comment (clacote) [inline src/main/resources/data/blog.json]: MiXiT 2017 **tickets** pre-registration

File: src/main/resources/data/blog.json [hunk 4]
```diff
@@ -1,5 +1,47 @@
...
+          "FRENCH": "Pre-enregistrement MiXiT"
```
Comment (clacote) [inline src/main/resources/data/blog.json]: Billetterie MiXiT 2017&nbsp;: pré-inscription ?

File: src/main/resources/messages_en.properties [hunk 5]
```diff
@@ -89,13 +89,13 @@ about.story2=Even if this conference has been created by Lyon JUG and CARA, you
 about.story3=Like each year, we are proud to welcome you for some expert sessions, games and meetings. Entrepreneurs, project managers, developers, designers, you will find ideas for your project! Talks are given in French and in English.
 about.teamtitle=The team
 
-attend.title=Attend
```
Comment (clacote) [inline src/main/resources/messages_en.properties]: I would find `Tickets` more obvious

**Comment by sdeleuze** [comment]
I am going to merge and add an additional commit with most of @clacote remarks taken in account (Slug are expected to be generated automatically based on the title)

**Pull request merged**